src/module/tests/hooks/MatrixHooks.ts

```diff
@@ -8,7 +8,12 @@ import { SuccessTest } from '../SuccessTest';
 export const MatrixHooks = {
     registerHooks: function () {
         Hooks.on('sr5_testPrepareBaseValues', MatrixHooks.onTestPrepareBaseValues_AddMatrixModifiers.bind(this));
+        // add extra damage to attack_matrix actions
```

Comment:
The handler function name already very cleanly tells this comments story :) So these three comments can likely be removed without loosing information.